.github/workflows/nanovdb.yml

```diff
@@ -153,21 +158,21 @@ jobs:
     strategy:
       matrix:
         config:
-          - { runner: 'macos-11', cxx: 'clang++', build: 'Release' }
-          - { runner: 'macos-11', cxx: 'clang++', build: 'Debug'   }
+          - { runner: 'macos-12', cxx: 'clang++', build: 'Release' }
+          - { runner: 'macos-12', cxx: 'clang++', build: 'Debug'   }
       fail-fast: false
     steps:
       - uses: actions/checkout@v3
       - name: install_deps
         run: ./ci/install_macos.sh
       - name: build
         run: >
-          ./ci/build.sh -v
+          sudo ./ci/build.sh -v
```

Comment:
Why do we need sudo here?

Reply:
The error that I believe this is trying to fix is this error which happens very early in the macos-build job:
```
getopt: illegal option -- l
getopt: illegal option -- g
```

This was happening to me on my Mac when trying to test out this action.  This is probably happening because the bash script is picking up the system's getopt and not the brew-installed `gnu-getopt` package installed binary.  The brew-installed `getopt` is attempting to be path'd here in the `.ci/install_macos.sh`

https://github.com/AcademySoftwareFoundation/openvdb/blob/6719224567c11211df7b239c13fa0025ae2a062c/ci/install_macos.sh#L34

But that doesn't seem to be working if we're getting this getopt error.

Reply:
I believe that this `getopt` issue is also the source of the problems with the `macos-houdini` CI jobs because the line crashing the build is also a `getopt` command